### PR TITLE
Add checksum for py-networkx 2.8.6, py-pygraphviz 1.10

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
@spackbot fix style

File: var/spack/repos/builtin/packages/py-networkx/package.py [hunk 1]
```diff
@@ -24,16 +25,32 @@ class PyNetworkx(PythonPackage):
...
+
     depends_on("python@2.7:", type=("build", "run"))
-    depends_on("python@3.5:", type=("build", "run"), when="@2.3:")
-    depends_on("python@3.6:", type=("build", "run"), when="@2.5:")
-    depends_on("python@3.7:", type=("build", "run"), when="@2.6:")
-    depends_on("python@3.8:", type=("build", "run"), when="@2.7:")
+    depends_on("python@3.5:", when="@2.3:", type=("build", "run"))
+    depends_on("python@3.6:", when="@2.5:", type=("build", "run"))
+    depends_on("python@3.7:", when="@2.6:", type=("build", "run"))
+    depends_on("python@3.8:", when="@2.7:", type=("build", "run"))
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-networkx/package.py:38]: Thanks.  That reads better IMHO.

File: var/spack/repos/builtin/packages/py-pygraphviz/package.py [hunk 2]
```diff
@@ -14,8 +14,10 @@ class PyPygraphviz(PythonPackage):
 
     maintainers = ["haralmha"]
 
+    version("1.10", sha256="457e093a888128903251a266a8cc16b4ba93f3f6334b3ebfed92c7471a74d867")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pygraphviz/package.py:17]: Confirmed sha256.

File: var/spack/repos/builtin/packages/py-networkx/package.py [hunk 3]
```diff
@@ -13,6 +13,7 @@ class PyNetworkx(PythonPackage):
     homepage = "https://networkx.github.io/"
     pypi = "networkx/networkx-2.4.tar.gz"
 
+    version("2.8.6", sha256="bd2b7730300860cbd2dafe8e5af89ff5c9a65c3975b352799d87a6238b4301a6")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-networkx/package.py:16]: Confirmed sha256.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**